shared/settings/account/confirm-delete.tsx

```diff
@@ -20,8 +21,9 @@ const getIcon = (props: Props) => {
   return Styles.isMobile ? 'icon-phone-number-remove-64' : 'icon-phone-number-remove-48'
 }
 const getPrompt = (props: Props) => {
-  const addr = props.type === 'email' ? props.address : PhoneUtil.e164ToDisplay(props.address)
-  return `Delete ${props.type === 'email' ? 'address' : 'number'} \n${addr}?`
+  return props.type === 'email'
+    ? `Delete email ${props.address}`
```

Comment:
no newline?

Reply:
Oh, I'm ambivalent about it - I didn't notice that the newlines in your suggestion are in the strings. I feel like it might look odd with the newline with a short email address, but if you feel strongly I can add them.

Reply:
i don't care, it should match the design. just pointing out it mutated from the original

Reply:
(I see - fixed to match design)